bot.py

```diff
@@ -0,0 +1,91 @@
+import os
+import platform
+import contextlib
+import hashids
+import datetime
+from google.cloud import datastore
+
+import discord
+from discord.ext import commands
+from discord import option
+from dotenv import load_dotenv
+
+import scanner
+load_dotenv()
```

Comment:
I don't think dotenv is a built in library? Do we need to add it to the requirements?

Maybe there's a simple built in solution we can use like json/configparser? or maybe even a simple python file like constants.py